interface/overview.rst

```diff
@@ -53,14 +54,14 @@ All Rock-ons require the **Rock-on service** to be enabled and prior to
 enabling this service it must be configured. This is a simple matter of
 configuring a sufficiently large share for the rock-ons to be installed into.
 
-We recommend creating a *dedicated* share to be used as the rock-ons root.
-While any share can be chosen as the rock-ons root, we **strongly** recommend
-to create a share to be used *only* as the rock-ons root to prevent any
-conflict that may arise with a mixed-used share. The rock-ons root share is
+It is recommended to create a *dedicated* share to be used as the Rock-ons root.
+While any share can be chosen as the Rock-ons root, it is **strongly** recommended
+to create a share to be used **only** as the Rock-ons root to prevent any
+conflict that may arise with a mixed-used share. The Rock-ons root share is
 used by Docker to store permanent data such as images and container
 layers. As a result, any conflict or alteration of these data that might result
 from a mixed used share has the potential to break the installed Rock-ons.
```

Comment:
```suggestion
from a mixed-use share has the potential to break the installed Rock-ons.
```

Again this does not pertain to your alterations but we could slip this in as we go.